canonical-web-and-design/maas-ui
# Change navigation hash links to links

## Done
Changed the hashlinks in the sidebar nav of settings to real links

## QA
Hover over the links and see that they have full paths rather than hashes
The links will all redirect to `/configuration` unless a route is defined in `ui/src/app/settings/components/Routes/Routes.js`
